lightspeed/modules/proc_monitor-onpremise-deployment.adoc

```diff
@@ -2,25 +2,37 @@
 
 [id="monitor-lightspeed-onpremise-deployment_{context}"]
 
-= Monitoring {LightspeedShortName} on-premise deployments
+= Monitoring your {LightspeedShortName} on-premise deployment
 
-After the {LightspeedShortName} on-premise deployment is successful, use the following procedure to monitor the metrics on an API endpoint `/api/v2/metrics`. 
-
-.Prerequisites
-* You have set up a {LightspeedShortName} on-premise deployment.
+After the {LightspeedShortName} on-premise deployment is successful, use the following procedure to monitor the metrics on an API endpoint `/metrics`. 
 
 .Procedure
 
 . Create a *system auditor* user:
-.. Create a user with the *system auditor* role in the the {PlatformName}. Refer to the procedure in the topic link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_user_guide/index#proc-controller-creating-a-user[Creating a user] in the _Automation Controller User Guide_.
+.. Create a user with a *system auditor* role in the the {PlatformName}. Refer to the procedure in the topic link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/automation_controller_user_guide/index#proc-controller-creating-a-user[Creating a user] in the _Automation Controller User Guide_.
```

Comment:
FYI: In 2.5 the Creating a user information will move to the new Access management and authentication draft. You'll want to note that this links will need to be updated after 2.5 is published.

Reply:
Yes, I have an action item to update all existing links to point to 2.5 branch once it's published. Thank you for the reminder.